sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations/VirtualMachineUpdate.cs

```diff
@@ -12,7 +12,71 @@ namespace Microsoft.Azure.Management.Compute.Models
+        public VirtualMachineUpdate(IDictionary<string, string> tags = default(IDictionary<string, string>), Plan plan = default(Plan), HardwareProfile hardwareProfile = default(HardwareProfile), StorageProfile storageProfile = default(StorageProfile), AdditionalCapabilities additionalCapabilities = default(AdditionalCapabilities), OSProfile osProfile = default(OSProfile), NetworkProfile networkProfile = default(NetworkProfile), SecurityProfile securityProfile = default(SecurityProfile), DiagnosticsProfile diagnosticsProfile = default(DiagnosticsProfile), SubResource availabilitySet = default(SubResource), SubResource virtualMachineScaleSet = default(SubResource), SubResource proximityPlacementGroup = default(SubResource), string priority = default(string), string evictionPolicy = default(string), BillingProfile billingProfile = default(BillingProfile), SubResource host = default(SubResource), SubResource hostGroup = default(SubResource), string provisioningState = default(string), VirtualMachineInstanceView instanceView = default(VirtualMachineInstanceView), string licenseType = default(string), string vmId = default(string), string extensionsTimeBudget = default(string), int? platformFaultDomain = default(int?), ScheduledEventsProfile scheduledEventsProfile = default(ScheduledEventsProfile), string userData = default(string), CapacityReservationProfile capacityReservation = default(CapacityReservationProfile), VirtualMachineIdentity identity = default(VirtualMachineIdentity), IList<string> zones = default(IList<string>))
```

Comment:
Maybe we can move `CapacityReservationProfile capacityReservation = default(CapacityReservationProfile),` to the end so that the new method is compatible with the old one?

Reply:
@archerzz , from what we know, we cannot change the order of the method parameters. And we certainly can't change them in the files in the Generated folders as those changes will be undone next time the SDK code is generated for a release. That change is not tenable long term. 

Since the method is having a new optional parameter inputted into its signature and we cannot change that, we need to ensure each combination of method parameters (that were moved by the new parameter) that can be inputted will be inputted to the correct parameter.

Reply:
this cannot be moved because it is generated from swagger and will be overwritten with each generation.
customization/VirtualMachineUpdate.cs edited to preserve original signature.